modules/ipi-install-validation-checklist-for-nodes.adoc

```diff
@@ -8,19 +8,21 @@
 
 .When using the `provisioning` network
 
-* [ ] NIC1 VLAN is configured for the `provisioning` network. (optional)
-* [ ] NIC1 is PXE-enabled on the provisioner, control plane (master), and worker nodes when using a `provisioning` network. (optional)
-* [ ] NIC2 VLAN is configured for the `baremetal` network.
+* [ ] (Optional) NIC VLAN is configured for the `provisioning` network.
+* [ ] (Optional) NIC for the `provisioning` network is PXE-enabled on the provisioner, control plane (master), and worker nodes.
+* [ ] NIC VLAN is configured for the `baremetal` network.
```

Comment:
Dropping the numerical identifiers for the NICs has made this ambiguous. We still need some way to differentiate the provisioning and baremetal NICs.